lib/apis/v3/felixconfig.go

```diff
@@ -214,6 +214,8 @@ type FelixConfigurationSpec struct {
 	DebugDisableLogDropping         *bool            `json:"debugDisableLogDropping,omitempty"`
 	DebugSimulateCalcGraphHangAfter *metav1.Duration `json:"debugSimulateCalcGraphHangAfter,omitempty" configv1timescale:"seconds"`
 	DebugSimulateDataplaneHangAfter *metav1.Duration `json:"debugSimulateDataplaneHangAfter,omitempty" configv1timescale:"seconds"`
+
+	IptablesNATOutgoingInterfaceFilter string `json:"iptablesNATOutgoingInterfaceFilter,omitempty" validate:"omitempty,natParam"`
```

Comment:
Suggest you change `natParam` to `ifaceFilter` since we could use that validation to validate other similar fields in future.

Reply:
ack